appengine/pom.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
not quite that easy for appengine unfortunately :) you have to package it as "war" and add appengine-plugin as dependency, etc: https://cloud.google.com/appengine/docs/java/tools/maven